### PR TITLE
move and fix tests

### PR DESCRIPTION
**Pull request by timse**
**What kind of change does this PR introduce?**

test bugfix

**Did you add tests for your changes?**

its a test

**If relevant, link to documentation update:**

N/A

**Summary**

fixes wrongly introduced tests in #4646 

**Does this PR introduce a breaking change?**

no

**Other information**

PTAL @sokra

### CODE REVIEW

File: test/identifier.test.js [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
-	describe("makeRelative", () => {
+describe("util/identifier", () => {
+	describe("makePathsRelative", () => {
 		describe("given a context and a pathConstruct", () => {
 			let context, pathConstruct, expected;
 			beforeEach(() => {
 				context = "/some/dir/";
 				pathConstruct = "/some/dir/to/somwhere|some/other/dir!../more/dir";
-				expected = `${path.relative(context, "/some/dir/to/somwhere")}|${path.relative(context, "some/other/dir")}!${path.relative(context, "../more/dir")}`;
+				expected = `${path.relative("/some/dir/", "/some/dir/to/somwhere")}|some/other/dir!../more/dir`;
```
Comment (timse) [inline test/identifier.test.js]: on windows `path.relative("/some/dir/", "/some/dir/to/somwhere")` becomes  `to\somwhere`.
thats not expected, is it? 
As this should yield the same identifier independent of OS?

Comment (sokra) [inline test/identifier.test.js]: It should yield a platform-independed identifier. Best replace `\` with `/` in `makePathsRelative`.

Comment (sokra) [inline test/identifier.test.js]: Best use a constant string for `expected` here

**Comment by sokra** [comment]
I'll merge this PR now, because circle ci is broken on master currently.

The fixes should be done in a separate PR.

**Pull request merged**